### PR TITLE
Qrcode doc correction

### PR DESCRIPTION
**Pull request by ankitdas13**
Doc reference :  https://razorpay.com/docs/api/qr-codes/

### CODE REVIEW

File: documents/qrcode.md [hunk 1]
```diff
@@ -373,6 +375,44 @@ $api->qrCode->fetch($qrCodeId)->close()
...
+
+$api->payment->fetch($paymentId)->refund($param);
+```
+
+**Parameters:**
+
+| Name            | Type    | Description                                                                  |
+|-----------------|---------|------------------------------------------------------------------------------|
+| payment*  | string | The id of the payment to be refunded |
+| param["amount"]  | string | Amount to be refunded |
```
Comment (sofiya02) [inline documents/qrcode.md:391]: change param[""] to amount and notes

File: documents/qrcode.md [hunk 2]
```diff
@@ -373,6 +375,44 @@ $api->qrCode->fetch($qrCodeId)->close()
 ```
 -------------------------------------------------------------------------------------------------------
 
+### Refund a Payment
+
+```php
+$param = array("amount" => "1000","notes" => array("purpose" => "Test UPI QR code notes"));
```
Comment (sofiya02) [inline documents/qrcode.md:381]: $api->payment->fetch($paymentId)->refund(array("amount"=> "100", "speed"=>"normal", "notes"=>array("notes_key_1"=>"Beam me up Scotty.", "notes_key_2"=>"Engage"), "receipt"=>"Receipt No. 31"));

**Review by sofiya02 — CHANGES_REQUESTED** [review]

File: documents/qrcode.md (hunk 2 above)
Comment (ankitdas13) [reply documents/qrcode.md]: updated

**Review by sofiya02 — CHANGES_REQUESTED** [review]
unresolved as per comment please recheck

**Comment by ankitdas13** [comment]
@sofiya02 use this payment id `pay_JccakgG2EZbHZO` for refund payment

**Review by sofiya02 — APPROVED** [review]
LGTM

**Pull request merged**